### DISCUSSION TITLE
Welcome to risco-lan-bridge Discussions!

### DISCUSSION THREAD

**Discussion by vanackej** [vanackej/risco-lan-bridge#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jatsoca** [comment]
[log stack prueba panel risco lightsys plus de risco.txt](https://github.com/user-attachments/files/23734005/log.stack.prueba.panel.risco.lightsys.plus.de.risco.txt)
Hola, he probado tu código y en paneles ProSYSPlus y funciona perfecto, pero ahora estoy probando en paneles risco LightSys Plus (RP432MP) con firmware FW 02.01.00.0004, pero en este panel a pesar de que conecta y trae la información del panel, perticiones y zonas, no trae las activaciones o eventos de zonas en tiempo real, aunque sí trae los eventos Trigger de las particiones en tiempo real. Cabe anotar que he modificado el código RicoComm para que deje conectar a paneles tipo RP432MP.
<img width="960" height="618" alt="image" src="https://github.com/user-attachments/assets/73914239-58aa-4fcf-a422-d95e30b33807" />
